src/CSET/_common_operators.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022-2024 Met Office and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Common operator functionality used across CSET.
+
+Functions below should only be added if it is not suitable as a standalone
+operator, and will be used across multiple operators.
+"""
+
+import iris
+import iris.cube
+
+
+def get_cube_xycoordname(cube: iris.cube.Cube, **kwargs) -> tuple:
```

Comment:
```suggestion
def get_cube_xycoordname(cube: iris.cube.Cube) -> tuple[str, str]:
```
Its only operators that need `**kwargs`, due to an implementation quirk.